versions/latest/modules/en/pages/k3k.adoc

```diff
@@ -0,0 +1,180 @@
+
+K3k integrates seamlessly with Rancher for simplified management of your embedded clusters.
+
+*Experimental Tool*
+
+This project is still under development and is considered experimental. It may have limitations, bugs, or changes. Please use with caution and report any issues you encounter. We appreciate your feedback as we continue to refine and improve this tool.
+
+== Features and Benefits
+
+* *Resource Isolation*: Ensure workload isolation and prevent resource contention between teams or applications. K3k allows you to define resource limits and quotas for each embedded cluster, guaranteeing that one team's workloads won't impact another's performance.
+
+* *Simplified Multi-Tenancy*: Easily create dedicated Kubernetes environments for different users or projects, simplifying access control and management. Provide each team with their own isolated cluster, complete with its own namespaces, RBAC, and resource quotas, without the complexity of managing multiple physical clusters.
+
+* *Lightweight and Fast*: Leverage the lightweight nature of K3s to spin up and tear down clusters quickly, accelerating development and testing cycles. Spin up a new K3k cluster in seconds, test your application in a clean environment, and tear it down just as quickly, streamlining your CI/CD pipeline.
+
+* *Optimized Resource Utilization (Shared Mode)*: Maximize your infrastructure investment by running multiple K3s clusters on the same physical host. K3k's shared mode allows you to efficiently share underlying resources, reducing overhead and minimizing costs.
+
+* *Complete Isolation (Virtual Mode)*: For enhanced security and isolation, K3k's virtual mode provides dedicated K3s server pods for each embedded cluster. This ensures complete separation of workloads and eliminates any potential resource contention or security risks.
+
+* *Rancher Integration*: Simplify the management of your K3k clusters with Rancher. Leverage Rancher's intuitive UI and powerful features to monitor, manage, and scale your embedded clusters with ease.
+
+== Installation
+
+This section provides instructions on how to install K3k and the k3kcli.
+
+=== Prerequisites
+
+* https://helm.sh/[Helm] must be installed to use the charts. Please refer to https://helm.sh/docs/[Helm's documentation] to get started.
+* An existing https://docs.rke2.io/install/quickstart[RKE2] Kubernetes cluster (recommended).
+* A configured storage provider with a default storage class.
+
+Note: If you do not have a storage provider, you can configure the cluster to use ephemeral or static storage. Please consult the https://github.com/rancher/k3k/blob/main/docs/advanced-usage.md#using-the-cli[k3kcli advance usage] for instructions on using these options.
```

Comment:
```suggestion
Note: If you do not have a storage provider, you can configure the cluster to use ephemeral or static storage. Please consult the xref:advanced-usage.adoc#_using_the_cli[k3kcli Advanced Usage] for instructions on using these options.
```

That file has been migrated to this repo so we shouldn't link to that k3k GitHub repo.